src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -883,25 +883,28 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                                 }
```

Comment:
@iSazonov, your last commit had 3 failures in `PowerShell-CI-windows`
Get-Error tests.Get-Error will accept pipeline input
```powershell
Expected case sensitive like wildcard '*-2146233087*' to match '
Exception             : 
    Type           : 
    ErrorRecord    : 
        Exception             : 
            Type    : 
            Message : 
            HResult : 
        CategoryInfo          : 
        FullyQualifiedErrorId : 
        InvocationInfo        : 
            ScriptLineNumber : 
            OffsetInLine     : 
            HistoryId        : 
            ScriptName       : 
            Line             : 
            PositionMessage  : 
            PSScriptRoot     : 
            PSCommandPath    : 
            CommandOrigin    : 
        ScriptStackTrace      : 
    TargetSite     : 
        Name          : 
        DeclaringType : 
        MemberType    : 
        Module        : 
    StackTrace     : 
   at System.Management.Automation.ExceptionHandlingOps.CheckActionPreference(FunctionContext funcContext, Exception exception)
   at System.Management.Automation.Interpreter.ActionCallInstruction`2.Run(InterpretedFrame frame)
   at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
   at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
    Message        : 
    Data           : 
    InnerException : 
        Type    : 
        Message : 
        HResult : 
    Source         : 
    HResult        : 
CategoryInfo          : 
FullyQualifiedErrorId : 
InvocationInfo        : 
    ScriptLineNumber : 
    OffsetInLine     : 
    HistoryId        : 
    ScriptName       : 
    Line             : 
    PositionMessage  : 
    PSScriptRoot     : 
    PSCommandPath    : 
    CommandOrigin    : 
ScriptStackTrace      : 

', but it did not match.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Get-Error.Tests.ps1: line 75
75:         $out | Should -BeLikeExactly '*-2146233087*'
```
Get-Error tests.Get-Error will handle Exceptions
```powershell
Expected case sensitive like wildcard '*myexception*' to match '
Type    : 
Message : 
HResult : 

', but it did not match.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Get-Error.Tests.ps1: line 83
83:         $out | Should -BeLikeExactly '*myexception*'
```
Get-Error tests.Get-Error adds ExceptionType for Exceptions
```powershell
Expected case sensitive like wildcard '*System.Management.Automation.ParentContainsErrorRecordException*' to match '
Exception             : 
    Type           : 
    ErrorRecord    : 
        Exception             : 
            Type    : 
            Message : 
            HResult : 
        CategoryInfo          : 
        FullyQualifiedErrorId : 
        InvocationInfo        : 
            ScriptLineNumber : 
            OffsetInLine     : 
            HistoryId        : 
            ScriptName       : 
            Line             : 
            PositionMessage  : 
            PSScriptRoot     : 
            PSCommandPath    : 
            CommandOrigin    : 
        ScriptStackTrace      : 
    TargetSite     : 
        Name          : 
        DeclaringType : 
        MemberType    : 
        Module        : 
    StackTrace     : 
   at System.Management.Automation.ExceptionHandlingOps.CheckActionPreference(FunctionContext funcContext, Exception exception)
   at System.Management.Automation.Interpreter.ActionCallInstruction`2.Run(InterpretedFrame frame)
   at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
   at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
    Message        : 
    Data           : 
    InnerException : 
        Type            : 
        Message         : 
        SocketErrorCode : 
        ErrorCode       : 
        NativeErrorCode : 
        TargetSite      : 
            Name          : 
            DeclaringType : 
            MemberType    : 
            Module        : 
        StackTrace      : 
   at System.Net.Dns.InternalGetHostByName(String hostName)
   at System.Net.Dns.GetHostByName(String hostName)
   at CallSite.Target(Closure , CallSite , Type , Object )
        Source          : 
        HResult         : 
    Source         : 
    HResult        : 
CategoryInfo          : 
FullyQualifiedErrorId : 
InvocationInfo        : 
    ScriptLineNumber : 
    OffsetInLine     : 
    HistoryId        : 
    ScriptName       : 
    Line             : 
    PositionMessage  : 
    PSScriptRoot     : 
    PSCommandPath    : 
    CommandOrigin    : 
ScriptStackTrace      : 

', but it did not match.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Get-Error.Tests.ps1: line 126
126:         $out | Should -BeLikeExactly "*$expectedExceptionType*"
```